components/contact/ContactIntro.jsx

```diff
@@ -23,7 +27,7 @@ function ContactIntro() {
           </Typography>
         </Box>
         <div className={classes.marginTopSmall}>
-          <Typography variant="body1">
+          <Typography variant="body1" className={classes.contentSentence}>
```

Comment:
can you add a center text alignment to this?
![image](https://github.com/hackforla/311-data/assets/8981287/f5931645-268c-46ba-8a3a-4c70c75104d1)

Reply:
Done